src/app/design/styles/color/index.html

```diff
@@ -355,6 +355,156 @@
 	</div>
 		
 	<div>
+  <stache-page-anchor>
+    Containers
+  </stache-page-anchor>
+    <p>
+    Use container colors when categorizing blocks of content that will be mixed together, such as a calendar with different types of events. Container colors can be chosen specifically by using their variable name, or applied in a sequence.
```

Comment:
It's minor, but I'd tighten up "when categorizing blocks of content that will be mixed together" to "to categorize blocks of content that mix together." ... I'd also change to "such" as clause to make it a separate sentence: "For example, use container colors for different types of events in calendars."

In the last sentence, we don't need the comma. Also, it's minor, but I'd tighten up "specifically by using their variable name, " to "by variable name"